augur/validate.py

```diff
@@ -67,45 +68,59 @@ def validate(jsonToValidate, schema):
                 trace = ["..."] + trace[-5:]
             trace = [str(x) for x in trace]
             print("\tERROR: {}. Trace: {}".format(error.message, " - ".join(trace)), file=sys.stderr)
-        raise ValidateError("Validation failed.")
-    else:
-        print("Validation succeeded")
+        raise ValidateError("Validation of {!r} failed.".format(filename))
 
 def auspice_config_v2(config_json, **kwargs):
     schema = load_json_schema("schema-auspice-config-v2.json")
-    jsonToValidate = load_json(config_json) if isinstance(config_json, str) else config_json
-    validate(jsonToValidate, schema)
+    if isinstance(config_json, str):
+        validate(load_json(config_json), schema, config_json)
+    else:
+        validate(config_json, schema, "<config>")
```

Comment:
This line should be changed to pass the filename of the config being validated (which is what you do for the other files being validated). This will result in the filename printed as output, rather than `Validating schema of '<config>'...`

Reply:
When `config_json` isn't a string, it's a dictionary or list object passed directly and there's no filename associated with it. Or am I misunderstanding?

Reply:
You're correct that `auspice_config_v2`, when `config_json` is a dict, doesn't currently know the filename from which the config was read, so my changes would necessitate changing the signature of `auspice_config_v2` to include this as a kwarg (or similar), and to add this argument to `get_config` within `export_v2.py`.

Reply:
Pushed a commit which removes the polymorphism of `config_json` in this function so the filename is always available.